### ISSUE TITLE
Creating a subfolder with compendia messes up compendia list

### ISSUE THREAD

**Issue by Tommycore** [earlSt1/vtt-compendium-folders#123]
**Describe the bug**
When trying to add a subfolder in the compendium tab, and selecting existing compendia from a module, those compendia get shot into nirvana.

**To Reproduce**
In a world with existing compendia from a module:
1. Open compendium tab and click on add folder.
1. In the dialog, choose some compendia that are in the folder where you click 'add folder'. Click 'Create Folder'.
1. Some of the chosen folders vanish, some show up at the very end as if belonging to no folder.
1. An error is thrown:
```
TypeError: this.collection.insert is not a function
    at CompendiumFolder.save (compendium-folders.js:196)
    at CompendiumFolder.addCompendium (compendium-folders.js:261)
    at async CompendiumFolderEditConfig._updateObject (compendium-folders.js:1316)
    at async CompendiumFolderEditConfig._onSubmit (foundry.js:3698)
```

**Expected behavior**
A new subfolder with the chosen items gets created.

**Screenshots**
### 1
![image](https://user-images.githubusercontent.com/19580006/145865466-9fccfa0d-932c-48db-925c-e94ee208721c.png)
### 2
![image](https://user-images.githubusercontent.com/19580006/145865529-52a9e0f7-df2f-42ce-9d5d-64e4a7ca90bb.png)
### 3
![image](https://user-images.githubusercontent.com/19580006/145865612-76e0e558-ce91-4c79-b349-6bc92f4ad71a.png)
### 4
![image](https://user-images.githubusercontent.com/19580006/145865658-7140382c-69a4-45dd-aa4b-bd57a1c93279.png)

**Information**
- FoundryVTT Version 9.235
- Compendium Folders Version 2.4.0
- Game System + Version HeXXen 0.8.0
- All other modules disabled? Active modules are libwrapper 1.11.0.0 and a content module relying on Compendium Folders.

**Comment by earlSt1** [comment]
Can you try again by uninstalling your current version and installing via this manifest? https://raw.githubusercontent.com/earlSt1/vtt-compendium-folders/09x-update/test/module.json

**Comment by earlSt1** [comment]
Fixed in v2.4.1
